plugins/chat/app/queries/chat/messages_query.rb

```diff
@@ -3,10 +3,10 @@
 module Chat
   # Queries messages for a specific channel. This can be used in two modes:
   #
-  # 1. Query messages around a target_message_id. This is used when a user
-  #    needs to jump to the middle of a messages stream or load around their
-  #    last read message ID. There is no pagination or direction here, just
-  #    a limit on past and future messages.
+  # 1. Query messages around a target_message_id or target_date. This is used
+  #    when a user needs to jump to the middle of a messages stream or load
+  #    around their last read message ID. There is no pagination or direction
```

Comment:
Let's talk of target after, so we don't have to repeat message ID or date each time

```suggestion
  #    around a target. There is no pagination or direction
```